pkg/pool-weighted/contracts/WeightedPoolProtocolFees.sol

```diff
@@ -238,7 +238,7 @@ abstract contract WeightedPoolProtocolFees is BaseWeightedPool, ProtocolFeeCache
         uint256[] memory normalizedWeights,
         uint256 preJoinExitSupply,
         uint256 postJoinExitSupply
-    ) internal view returns (uint256, uint256) {
+    ) internal returns (uint256) {
```

Comment:
Would loudly document the side effect here; it's not a pure getter. (Same for yield.)